.eslintrc.js

```diff
@@ -25,5 +25,13 @@ module.exports = {
     'react-hooks/exhaustive-deps': WARNING,
     'react-hooks/rules-of-hooks': ERROR,
     'react/jsx-no-bind': [IGNORE]
-  }
+  },
+  overrides: [
+    {
+      files: ['packages/site/pages/**/*.js', '**/__stories__/*.js'],
+      rules: {
+        'react/jsx-key': [IGNORE]
```

Comment:
why ignore this rule in site and stories?

Reply:
Because there are 100+ places where  key has not been used. I can go through and refactor each but from now on we will have to apply a key when writing stories. Thoughts?

Reply:
WARNING is probably better than ignore

Reply:
Edward, it sounds like we have lint debt in the stories and now we're turning off a rule because keeping the rule isn't worth it to change the 100+ places.  If that's the case, should we be turning off eslint generally for stories or removing this rule specifically from the greater codebase (If the ROI on the rule or the code linted isn't in our favor.)?

Reply:
Will go with WARNING for now